src/include/cursor.h

```diff
@@ -211,33 +211,30 @@ struct __wt_cursor_btree {
 struct __wt_cursor_bulk {
 	WT_CURSOR_BTREE cbt;
 
-	WT_REF	*ref;			/* The leaf page */
-	WT_PAGE *leaf;
```

Comment:
Was there a reason why you moved these lower in the structure? I quite liked that the shared portions of the bulk cursor were at the top. I don't care much, just wondering.